lib/utils/search.js

```diff
@@ -80,4 +80,33 @@ const isJSON = text => {
   return true;
 };
 
-export { getSearchResults };
+const getWordsToHighlight = searchTerm => {
+  const synonyms = isJSON(process.env.SYNONYMS) ? JSON.parse(process.env.SYNONYMS) : '{}';
+
+  let normalisedSearchTerm = normalise(searchTerm.trim());
+  let highlightables = removeCommonWords(normalisedSearchTerm.split(' '));
```

Comment:
I would probably call this wordsToHighlight or highlightableWords, or just 'words' may be clear enough in the context of the method.

Highlightables might be confusing and initially I saw the word 'tables' :-)